src/types/Ctype.ts

```diff
@@ -13,8 +15,9 @@ export interface ICTypeInput {
   properties: object[] // TO DO: need to refine what properties are
```

Comment:
I don't think this should be an Array of objects?

Reply:
not part of this ticket